Dockerfile

```diff
@@ -108,15 +108,15 @@ ARG CACHEBUST
 RUN luet install -y \
     toolchain/yip \
     toolchain/luet \
-    utils/installer@0.18 \
+    utils/installer@0.21 \
     system/cos-setup \
-    system/immutable-rootfs@0.2.0-11 \
+    system/immutable-rootfs@0.2.0-12 \
     system/grub2-config \
     selinux/k3s \
     selinux/rancher \
     utils/k9s \
     utils/nerdctl \
-    utils/rancherd@0.0.1-alpha07-9 \
+    utils/rancherd@0.0.1-alpha11-2 \
```

Comment:
This will break the installation.
To bump rancherd, we need the works in this draft PR: https://github.com/harvester/harvester-installer/pull/146

Reply:
Is https://github.com/harvester/harvester-installer/pull/146 ready to be tested (it's still in Draft state)? I could help test it if you need.

Reply:
I'll rebase it, thanks

Reply:
Since it requires extra effort to upgrade `rancherd`, we decided to pin down the version of `rancherd`. Because Luet no longer provides that version from the current index, we need to manually pull the binary from GitHub.

Reply:
Let's pin to the latest index after https://github.com/harvester/harvester-installer/pull/146 is done.